File: main/src/ca/uwaterloo/flix/runtime/solver/api/predicate/FalsePredicate.scala
package ca.uwaterloo.flix.runtime.solver.api.predicate

/**
  * Represents the false predicate.
  */
class FalsePredicate extends Predicate {
  /**
    * Returns a string representation of `this` predicate.
    */
  override def toString: String = "false"
}

File: main/src/ca/uwaterloo/flix/runtime/solver/api/predicate/FalsePredicate.java
package ca.uwaterloo.flix.runtime.solver.api.predicate;

/**
 * Represents the false predicate.
 */
public final class FalsePredicate implements Predicate {

    /**
     * Returns a human-readable representation of `this` predicate.
     */
    @Override
    public String toString() {
        return "false.";
    }

}
